app/assets/javascripts/hyku_addons/autofill_work_form.js

```diff
@@ -76,16 +90,16 @@ class AutofillWorkForm {
       })
 
     } else if ($.type(value) == "object") {
-      var wrapper = this.wrapper(field, index)
+      var $wrapper = this.wrapper(field, index)
 
       // This isn't using the normal setValue method because of the requirement to autofill nested groups of fields
       Object.entries(value).forEach(([childField, childValue]) => {
-        $(wrapper.find($(this.inputSelector(childField))).find(this.targetInputSelector).get(0)).val(childValue)
+        $($wrapper.find($(this.inputSelector(childField))).find(this.targetInputSelector).get(0)).val(childValue)
```

Comment:
This gets confusing to read / understand

Reply:
Yea, its a little scruffy. I'll tidy that up now.